src/components/pricing/select-plan-new/index.tsx

```diff
@@ -48,10 +54,15 @@ const PlanQuantitySelect: React.FunctionComponent<{
           value={quantity}
           max={1000}
           min={1}
-          onChange={(e) => setQuantity(Number(e.currentTarget.value))}
+          onChange={(e) => onQuantityChanged(Number(e.currentTarget.value))}
         />
       </label>
-    </>
+      {quantity > 1 && (
```

Comment:
@vojtaholik this is showing them how much each seat costs for reference. Wasn't super attractive but functional